webapp/src/extensionsBrowser.tsx

```diff
@@ -33,31 +33,19 @@ export const ExtensionsBrowser = (props: ExtensionsProps) => {
 
     const [searchFor, setSearchFor] = useState("");
     const [allExtensions, setAllExtensions] = useState(fetchBundled());
-    const [extensionsToShow, setExtensionsToShow] = useState([]);
+    const [extensionsToShow, setExtensionsToShow] = useState(new Array<ExtensionMeta & EmptyCard>());
```

Comment:
in general, no one really uses array constructors in js. if you wan to hint the type, you can do it like this
```suggestion
    const [extensionsToShow, setExtensionsToShow] = useState<(ExtensionMeta & EmptyCard)[]>([]);
```

Reply:
Ah, sweet! I definitely was aiming for just knowing the type so that when referencing it later on I could see its members.